modules/UI/videolayout/RemoteVideo.js

```diff
@@ -164,7 +163,9 @@ RemoteVideo.prototype._generatePopupContent = function () {
     let remoteControlState = null;
     let onRemoteControlToggle;
 
-    if (this._supportsRemoteControl) {
+    if (this._supportsRemoteControl
+        && ((!APP.remoteControl.active && !this._isRemoteControlSessionActive)
+            || APP.remoteControl.controller.activeParticipant === this.id)) {
```

Comment:
It should be possible. As far as I know there isn't anything related to the remote control in redux.